### PR TITLE
[DA-668] ETL scripts: add cli flags to set paths, removed hardcoded paths

### PR DESCRIPTION
**Pull request by robabram**
Added '--local' flag to mysqlimport statements to fix error accessing local files.

### CODE REVIEW

**Comment by robabram** [comment]
I hadn't used them until this week. I saw a ticket pending to remove the hard coded paths, so I updated them.

File: rest-api/etl/setup_database_for_etl.sh [hunk 1]
```diff
@@ -6,29 +6,40 @@
 # If the databases already exist, drop them prior to running this.
 # Note: this takes a while. Go get some coffee while it's running!
 
-USAGE="tools/setup_database_for_etl.sh --project <PROJECT> --account <ACCOUNT>"
```
Comment (Seeker1911) [inline rest-api/etl/setup_database_for_etl.sh]: All tools assume they're being ran from rest-api/ 
For instance, this one sources `tools/auth_setup.sh` on L43 which will fail.
I'm all for the shortest distance being a straight line, but not sure we should change established patterns, this one would require quite a bit of work to establish across the board. I recommend staying with the pattern on this.

File: rest-api/etl/setup_local_database_for_etl.sh [hunk 2]
```diff
@@ -23,27 +23,48 @@ DB_CONNECTION_NAME=
 CSV_DIR=/var/lib/mysql-files/rdr-csv
 OUTPUT_DIR=/tmp/rdr-sql-dump
 
-USAGE="tools/setup_local_database_for_etl.sh --account <ACCOUNT> [--generate_sql_dump] [--db_user <ROOT_DB_USER>] [--nopassword]"
```
Comment (Seeker1911) [inline rest-api/etl/setup_local_database_for_etl.sh]: same as above.

File: rest-api/etl/setup_database_for_etl.sh (hunk 1 above)
Comment (robabram) [reply rest-api/etl/setup_database_for_etl.sh]: No problem.  I've updated the usages to show the correct 'etl' directory.

**Pull request merged**